grid/app/main/sfl/cycles/cycle.py

```diff
@@ -14,15 +14,15 @@ class Cycle(db.Model):
         fl_process_id (Integer,ForeignKey): Federated learning ID that owns this cycle.
     """
 
-    __tablename__ = "__cycle__"
+    __tablename__ = "static_cycle_"
```

Comment:
Why are we keeping one `_` after table names?

Reply:
No particular reason, it simply wasn't specified that it should be removed.